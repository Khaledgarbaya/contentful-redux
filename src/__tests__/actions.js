const actions = require('../actions');

test('sync action creator', () => {
	expect(actions.sync()).toMatchSnapshot();
});
