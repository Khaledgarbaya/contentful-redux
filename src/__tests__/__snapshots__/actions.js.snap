// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sync action creator 1`] = `
Object {
  "type": "Contentful/SYNC",
}
`;
